src/test/java/org/mockitousage/MockitoAnnotationsTest.java

```diff
@@ -0,0 +1,89 @@
+package org.mockitousage;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.MockitoPlugin;
+import org.mockito.configuration.AnnotationEngine;
+import org.mockito.internal.configuration.GlobalConfiguration;
+import org.mockito.internal.util.MockUtil;
+import org.mockito.runners.MockitoJUnitRunner;
+
+public class MockitoAnnotationsTest {
+
+    @Before
+    public void setUp() {
+         GlobalConfiguration.removeIt();
+    }
+    
+    @After
+    public void tearDown() {
+         GlobalConfiguration.removeIt();
+    }
+
+    @Test
+    public void should_process_the_testcase_using_default_annotation_engine() throws Exception {
+        NoMockitoPluginAnnotationTestClass noMockitoPluginAnnotationTestClass = new NoMockitoPluginAnnotationTestClass(); 
+
+        MockitoAnnotations.initMocks(noMockitoPluginAnnotationTestClass);
+        assertThat(isMock(noMockitoPluginAnnotationTestClass.someMock), is(true));
```

Comment:
You may want to prefer AssertJ assertions, it's in our dependencies (and it replaces FEST Assert).